snowplow/iglu-client-embedded/schemas/com.metabase/instance_stats/jsonschema/1-0-1

```diff
@@ -0,0 +1,160 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for daily stats ping, tracking instance metrics and settings",
+  "self": {
+    "vendor": "com.metabase",
+    "name": "instance_stats",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "instance_attributes": {
+      "description": "Key-value pairs of instance attributes",
+      "type": "array",
+      "items": {
+        "type": "object",
+        "description": "A single instance attribute",
+        "properties": {
+          "key": {
+            "description": "The key for this attribute",
+            "type": "string",
+            "maxLength": 255
+          },
+          "value": {
+            "description": "The value of this attribute",
+            "type": ["string", "boolean", "integer", "null"],
+            "maxLength": 255,
+            "minimum": 0,
+            "maximum": 2147483647
+          }
+        },
+        "required": ["key", "value"]
+      }
+    },
+    "metrics": {
+      "description": "Key-value pairs of metrics",
+      "type": "array",
+      "items": {
+        "type": "object",
+        "description": "A single metric attribute",
+        "properties": {
+          "key": {
+            "description": "The key for this attribute",
+            "type": "string",
+            "maxLength": 255
+          },
+          "value": {
+            "description": "The value of this attribute",
+            "type": ["string", "boolean", "integer"],
+            "maxLength": 255,
+            "minimum": 0,
+            "maximum": 2147483647
+          },
+          "tags": {
+            "description": "Tags for this metric",
+            "type": "array",
+            "items": {
+              "type": "string",
+              "maxLength": 255
+            }
+          }
+        },
+        "required": ["key", "value"]
```

Comment:
```suggestion
          "key": {
            "description": "The unique name of the metric",
            "type": "string",
            "maxLength": 255
          },
          "value": {
            "description": "The value of the metric",
            "type": ["number"],
            "maxLength": 255,
            "minimum": 0,
            "maximum": 2147483647
          },,
          "tags": {
            "description": "Tags that can be used flagging teams / features the metric belongs to",
            "type": "array",
            "items": {
              "type": "string",
              "maxLength": 255
            }
          }
        },
        "required": ["name", "value"]
```

Reply:
This almost all looks fine, except that we need to remember that `cache_num_queries_cached` is actually a string value because it is a binned balue. We need to either change how it's computed or allow strings in here. (example strings are "0", "1-5", etc. from [here](https://github.com/metabase/metabase/blob/add-metrics-to-snowplow-ping/src/metabase/analytics/stats.clj#L54).

So, we could just return the number and not bin it.

Reply:
> `"required": ["name", "value"]`

The maps in this array have the keys: "key" "value" and "tags". I don't think we can require a "name" key w/o specifying what it is.

Reply:
I've made this into `"required": ["key", "value", "tags"]` since they will always have those keys.

Reply:
>So, we could just return the number and not bin it.

Yes agreed. If we had binned stuff and care about the counts per bin, this would more become a grouped metric. In this case it sounds reasonable to just always return numbers for metrics.

> The maps in this array have the keys: "key" "value" and "tags". I don't think we can require a "name" key w/o specifying what it is.

Ah yeah sorry - good catch. I forgot to change the `key` to `name`. I think in the schema from the docs, we agreed on `name` for metric instead of `key`.  Similar to how we do it for grouped metrics. There you also have a `name`.
Does that make more sense with that context?

Reply:
Cool, I will switch it to "name" "value" and "tags", and unbin `cache_num_wueries_cached`.